lib/fathead/beautifulsoup/parse.py

```diff
@@ -151,10 +151,10 @@ def create_file(self):
 
         with open('output.txt', 'w+') as output_file:
             for data_element in self.data:
-                name = data_element.get('name')
+                name = data_element.get('name').lower()
                 code = data_element.get('code')
                 first_paragraph = data_element.get('first_paragraph')
-                abstract = '{}{}{}'.format(code, '<br>' if code else '', first_paragraph)
+                abstract = '<section class="prog__container">{}{}{}</section>'.format(code, '<br>' if code else '', first_paragraph)
```

Comment:
we can remove the `<br>` here too -- they're not needed with the templates, as we've added padding around code blocks and `<p>`